lib/rules/no-unknown-arguments-for-builtin-components.js

```diff
@@ -278,6 +279,31 @@ function removeAtSymbol(txt) {
   return txt.replace('@', '');
 }
 
+function fix(node, argument) {
+  const tagMeta = KnownArguments[node.tag];
+  const deprecatedArgs = tagMeta.deprecatedArguments || {};
+  const replacementArg = deprecatedArgs[argument.name];
+  if (replacementArg) {
+    argument.name = replacementArg;
+    return true;
+  }
+  const deprecatedEvents = tagMeta.deprecatedEvents || {};
```

Comment:
Can you add a comment with an example conversion to each of the two parts of this `fix` function to make it clear to the reader what is going on?

Reply:
Oh, sorry, lost in notifications and forgot about it :(

Reply:
No problem, feel free to follow-up with improvements.